sunsetstation.dme

```diff
@@ -2708,6 +2708,7 @@
 #include "code\modules\vehicles\vehicle_key.dm"
 #include "code\modules\vehicles\cars\car.dm"
 #include "code\modules\vehicles\cars\clowncar.dm"
+#include "code\modules\vending\_vending.dm"
 #include "sunset\code\modules\vending\_vending.dm"

```

Comment:
Don't edit the DME manually!!!!!!!

Reply:
I had accidentally removed the original line for _vending so i just put it back so it looks like that in the commit history but was just a mistake..

Reply:
I'll merge this if you fix it. All you need to do to re-sort DME is to just click compile in Dream Maker. You don't even need to wait for it to compile. You can just compile, then immediately stop the compilation.

Reply:
Re-sorted the DME and thank you for the explanation and I'll make sure it's sorted in the future.